test/common.js

```diff
@@ -423,8 +423,10 @@ function runCallChecks(exitCode) {
 }
 
 
-exports.mustCall = function(fn, expected) {
-  if (typeof expected !== 'number') expected = 1;
+exports.mustCall = function(fn, expected = 1) {
```

Comment:
I don't think this syntax is supported on v4. This will need to be changed now or during backport.